docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -0,0 +1,854 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Getting started with Reactive
+
+include::./attributes.adoc[]
+
+:toc: macro
+:toclevels: 4
+:doctype: book
+:icons: font
```

Comment:
Please don't use that here, we want this config to be global and not per guide.

Reply:
I copied it from another file

Reply:
(the original getting-started)

Reply:
OK, I suppose it will work fine then.

Reply:
I've removed them, wondering if we should remove it from the original getting-started.

Reply:
I think it would be safer. If one day we want to change it, we will want to do it globally.